### PR TITLE
Added netcat Binary and changed the dockerfile

### PR DESCRIPTION
**Pull request by tommyd450**
Adding netat binary for hermetic Builds

### CODE REVIEW

File: redhat/overlays/Dockerfile.netcat [hunk 1]
```diff
@@ -1,3 +1,3 @@
 FROM registry.access.redhat.com/ubi9/ubi-minimal@sha256:9c810eccada0cc6f25b46eeeea4516bab01828faef4fdd2fe49c760518b90dd1
 
-RUN microdnf -y install nc
+COPY netcat /usr/bin
```
Comment (lance) [inline redhat/overlays/Dockerfile.netcat:3]: We need to think about this and perhaps consult with prodsec. I think having an executable of unknown provenance in the image might not work due to security concerns.

Comment (tommyd450) [reply redhat/overlays/Dockerfile.netcat:3]: That makes sense. We can just forgo the hermetic build on this image then for the time being just so we can have an built image?

Comment (lance) [reply redhat/overlays/Dockerfile.netcat:3]: Yeah - let's do that.

**Comment by lance** [comment]
@tommyd450 closing this for now

**Pull request closed without merging**